src/stores/commandStateStore.ts

```diff
@@ -20,12 +25,18 @@ export const resetCommandState = () => {
 
 /** Updates the command state to the current state of the document. */
 export const updateCommandState = () => {
-  commandStateStore.update({
-    bold: document.queryCommandState('bold'),
-    italic: document.queryCommandState('italic'),
-    underline: document.queryCommandState('underline'),
-    strikethrough: document.queryCommandState('strikethrough'),
-  })
+  const state = store.getState()
+  if (!state.cursor) return
+  const isActive = selection.isActive()
+  const action = isActive
```

Comment:
Let's just inline `selection.isActive()`. It's not really necessary to have a separate variable.

Reply:
Sounds good to me. Done: 92f25c9ae49aa433807c92151e894d169806648d